appengine/flexible/tasks/pull_queues/README.md

```diff
@@ -0,0 +1,94 @@
+# Google Cloud Tasks Pull Queue Samples
```

Comment:
Since this is a command-line sample, can you use the readme generator? (see `storage/cloud-client/README.rst.in` for an example.)

Reply:
Okay, will have to look into how that works, I'll add it to the list.